taxi/forms.py

```diff
@@ -0,0 +1,35 @@
+from django import forms
+from django.contrib.auth.forms import UserCreationForm
+
+from taxi.models import Driver, Car
+
+
+class DriverCreationForm(UserCreationForm):
+    class Meta(UserCreationForm.Meta):
+        model = Driver
+        fields = UserCreationForm.Meta.fields + (
+            "first_name", "last_name", "license_number",
+        )
+
+
+class DriverDeleteForm(forms.ModelForm):
+    class Meta:
+        model = Driver
+        fields = []
+
+
+class DriverLicenseUpdateForm(forms.ModelForm):
+    class Meta:
+        model = Driver
+        fields = [
+            "username", "first_name", "last_name", "license_number",
+        ]
+
+
+class CarsCreatedForm(forms.ModelForm):
+    class Meta:
+        model = Car
+        fields = "__all__"
+        widgets = {
+            "drivers": forms.CheckboxSelectMultiple()
```

Comment:
Ensure that the `drivers` field in the `Car` model is a `ManyToManyField` or similar, as `CheckboxSelectMultiple` is typically used for fields that can have multiple selections.